### PR TITLE
[WIP] get latest block test

### PR DESCRIPTION
**Pull request by neilZon**
Closes: #330 

## Description

This PR tests internal cmds 
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: internal/blocks/blocks_test.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+			Blocks: &MockBlocks{},
+		}
+
+		readerWriter := tests.ReaderWriter()
+		res, err := get([]string{"latest"}, readerWriter, command.Flags, s)
+		if err != nil {
+			panic(err)
+		}
+
+		expected := "Block ID\t6162000000000000000000000000000000000000000000000000000000000000\nParent ID\t6162000000000000000000000000000000000000000000000000000000000000\nTimestamp\t2022-07-27 14:27:49.585751 -0600 MDT m=+0.025544111\nHeight\t123\nTotal Seals\t0\nTotal Collections\t0"
```
Comment (sideninja) [inline internal/blocks/blocks_test.go:54]: first, you should assert the values in the result struct. The String comparison is a bit fragile, so I would say it would be better to have some kind of regex maybe to detect values in there.

File: pkg/flowkit/services/blocks.go [hunk 2]
```diff
@@ -50,6 +50,15 @@ func NewBlocks(
 	}
 }
 
+type IBlocks interface {
```
Comment (sideninja) [inline pkg/flowkit/services/blocks.go:53]: Uff you from Java? :D 
I think it's a good idea to make an interface for services, I think it would make sense to even have a single interface in front of all services, I don't like how I designed this a while ago. It would however be a big breaking change so I think it should be as a separate PR that then allows this tests PR.

Comment (neilZon) [reply pkg/flowkit/services/blocks.go:53]: > Uff you from Java? :D

Oh no is that a Javaism I'll come up with a better name 🤣
I'm having troubles thinking of a way to refactor the `Services` struct to be be defined by a single interface. Here is what I could think of https://github.com/onflow/flow-cli/compare/neil/refactor-services-for-testing

Comment (sideninja) [reply pkg/flowkit/services/blocks.go:53]: I think I would rather refactor that the whole service layer has one interface and then methods like `GetTransaction`, `GetBlock` etc. I think we should first discuss this in an issue, so open an issue, write your proposal, I will leave comments and lets define things first there and then move on with implementation. The issue can be generic "refactor of services to use an interface" not just for testing, since this will improve other things as well.

**Comment by sideninja** [comment]
Closing this in favour of defining first things in an issue.
